src/app/Components/LoadFailureView.tsx

```diff
@@ -7,9 +7,14 @@ import { ReloadIcon } from "../../palette/svgs/ReloadIcon"
 interface LoadFailureViewProps {
   error?: Error
   onRetry?: () => void
+  displayInBottom?: boolean
 }
 
-export const LoadFailureView: React.FC<LoadFailureViewProps> = ({ error, onRetry }) => {
+export const LoadFailureView: React.FC<LoadFailureViewProps> = ({
+  error,
+  onRetry,
+  displayInBottom,
```

Comment:
I'm not sure if "display below" is grammatically correct. What do you think about renaming it to `placeAtBottom` or just pass `justifyContent` as a prop?

```suggestion
  /**
   * Places the content at the bottom of the screen.
   */
  placeAtBottom,
```

or 

```suggestion
  justifyContent,
```

Reply:
Passing `justifyContent` as a prop is a better idea. I will change it. Thanks